tfhe/examples/utilities/print_doc_bench_parameters.rs

```diff
@@ -0,0 +1,84 @@
+use tfhe::keycache::NamedParam;
+use tfhe::shortint::parameters::classic::gaussian::p_fail_2_minus_64::ks_pbs::{
+    PARAM_MESSAGE_1_CARRY_1_KS_PBS_GAUSSIAN_2M64, PARAM_MESSAGE_2_CARRY_2_KS_PBS_GAUSSIAN_2M64,
```

Comment:
Here if you remove `_GAUSSIAN` from the name it'll select either Gaussian or TUniform depending on our default choices, I think it would be better.

Reply:
indeed, though we would be removing other precisions I think as we only have TUniform for the 2_2 use case IIRC